packages/headless/src/app/search-engine/search-engine-configuration.ts

```diff
@@ -1,10 +1,13 @@
-import {RecordValue, Schema} from '@coveo/bueno';
+import {ArrayValue, RecordValue, Schema} from '@coveo/bueno';
 import {
   PostprocessFacetSearchResponseMiddleware,
   PostprocessQuerySuggestResponseMiddleware,
   PostprocessSearchResponseMiddleware,
 } from '../../api/search/search-api-client-middleware';
-import {nonEmptyString} from '../../utils/validate-payload';
+import {
+  nonEmptyString,
+  requiredNonEmptyString,
+} from '../../utils/validate-payload';
```

Comment:
same as the other PR, `@utils` now available at shops near you!

Reply:
That's Headless though, it's not configured for this project, is it ? 🤔

Reply:
ah no, sorry! didn't think properly.. could do though

Reply:
You mean adding aliases to Headless ? Yeah it would be possible. But I think it's out of scope for this PR.